bindings/python/src/OpenSpaceToolkitPhysicsPy/Units/Derived/Angle.cpp

```diff
@@ -108,7 +109,7 @@ inline void OpenSpaceToolkitPhysicsPy_Units_Derived_Angle(pybind11::module& aMod
             arg("upper_bound")
         )
         .def("in_revolutions", &Angle::inRevolutions)
-        .def("to_string", &Angle::toString, arg("do_sanitize") = false)
+        .def("to_string", &Angle::toString, "aPrecision"_a = Integer::Undefined())
```

Comment:
use `arg` form instead to be in sync w/ other bindings, and `precision` instead of `aPrecision`

Reply:
Updated
However when doing so, i noticed that the arg name is wrong in the current bindings
I changed it from `aPrecision` to `precision`, but i think it will be difficult to detect where it's breaking code in other project because `to_string` is a very common name
Also added a unit test for that
What is your thinking @lucas-bremond @Derollez

Reply:
I think that should be okay, not a lot of notebooks use the toString representation for Angle, and the default precision is often used. I think you should be good to go!